libraries/matrix/impl/src/main/kotlin/io/element/android/libraries/matrix/impl/timeline/item/event/TimelineEventContentMapper.kt

```diff
@@ -103,7 +103,7 @@ class TimelineEventContentMapper(private val eventMessageMapper: EventMessageMap
             StickerContent(
                 body = kind.body,
                 info = kind.info.map(),
-                url = kind.url,
+                url = kind.source.url(),
```

Comment:
I think this PR https://github.com/element-hq/element-x-android/pull/2983 is also touching this area.